Update dnf command to add repo

inknos opened the pull request:
Since Fedora 41 dnf is replaced by dnf5 which uses an updated syntax and includes config-manager as a core command

almet commented:
Thanks I haven't noted the EOL date is past us :-)
Then things are different, you're right!

So, I do think we should remove support for F40 altogether, which means merging your changes and removing mentions of it in our docs.

We can start by merging your changes, and I've opened [a follow-up issue](https://github.com/freedomofpress/dangerzone/issues/1178) to track end of support for F40 in general.

inknos commented:
> One missing thing before merging is to update the CHANGELOG file and add an entry about your changes (we added this to ease the release workflow). I can also do it myself if you don't want to bother.

all right, @almet , I updated the changelog. let me know :)

almet commented:
Thanks @inknos!
